README.md

```diff
@@ -1,16 +1,106 @@
-# test #
+# skeleton-golang-package #
 
-[![GitHub Build Status](https://github.com/cisagov/test/workflows/build/badge.svg)](https://github.com/cisagov/test/actions)
+[![GitHub Build Status](https://github.com/cisagov/skeleton-golang-package/workflows/build/badge.svg)](https://github.com/cisagov/skeleton-golang-package/actions)
 
 This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) GitHub project started.
+new [cisagov](https://github.com/cisagov) Go project started.
 This skeleton project contains [licensing information](LICENSE), as
 well as [pre-commit hooks](https://pre-commit.com) and
 [GitHub Actions](https://github.com/features/actions) configurations
 appropriate for the major languages that we use.
 
-In many cases you will instead want to use one of the more specific
-skeleton projects derived from this one.
+## Installation ##
+
+To install the example package, use `go build`:
+
+```console
+cd ./skeleton-golang-package/src/example
+go build
+```
+
+This will produce an executable `example` within the current directory.
+
+Execute the `./example` binary to run the program.
+
+You should see the contents of the main package as defined in [main.go](src/example/main.go).
+
+### Nix ###
+
+If you have [Nix](https://nixos.org/download.html) installed you can use
+the [flake.nix](flake.nix) configuration file located at the root of the
+project to build and run the `example` package:
+
+```console
+cd ./skeleton-golang-package
```

Comment:
I think this line is redundant (based on the text that precedes it) and can be removed:
```suggestion
```